### PR TITLE
[14.0] shopinvader_quotation : fixes and improvements

### PR DESCRIPTION
**Pull request by ivantodorovich**
Superseeds https://github.com/shopinvader/odoo-shopinvader/pull/1110

### CODE REVIEW

File: shopinvader_quotation/services/quotation.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Copyright 2016 Akretion (http://www.akretion.com)
-# Copyright 2021 Camptocamp (http://www.camptocamp.com).
 # @author Benoît GUILLOT <benoit.guillot@akretion.com>
+# Copyright 2021 Camptocamp (http://www.camptocamp.com)
+# @author Simone Orsi <simone.orsi@camptocamp.com>
```
Comment (simahawk) [inline shopinvader_quotation/services/quotation.py:5]: and you?

**Review by simahawk — APPROVED** [review]

File: shopinvader_quotation/services/quotation.py (hunk 1 above)
Comment (ivantodorovich) [reply shopinvader_quotation/services/quotation.py:5]: 👋🏻

File: shopinvader_quotation/models/sale_order.py [hunk 2]
```diff
@@ -51,3 +51,12 @@ def action_request_quotation(self):
             if rec.shopinvader_backend_id:
                 rec.shopinvader_backend_id._send_notification("quotation_request", rec)
         return True
+
+    def write(self, vals):
+        # Set the typology to "quotation" when the quotation is sent.
+        # Normally there are two cases where this happens:
+        # * When the :meth:`action_quotation_sent` is called.
```
Comment (hparfr) [inline shopinvader_quotation/models/sale_order.py:58]: >When the **:meth:**`action_quotation_sent` is called

 :pill:  ?

File: shopinvader_quotation/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -3,7 +3,7 @@ This module adds a REST API for shopinvader to manage quotations.
 The Customer can convert a cart into a quotation (the typology of the sale
 order is set to quotation).
 
-Initialy, the quotation has the `shopinvader_state` "estimating".
+Initially, the quotation has the `shopinvader_state` "estimating".
 After updating the price manually when the button "sent" on Odoo backend
 is summited, the quotation will be sent by email (native behaviour) and the
```
Comment (hparfr) [inline shopinvader_quotation/readme/DESCRIPTION.rst]:
```suggestion
is submitted, the quotation will be sent by email (native behaviour) and the
```

Comment (ivantodorovich) [reply shopinvader_quotation/readme/DESCRIPTION.rst]: done! :)

**Pull request merged**